R/utils.R

```diff
@@ -290,7 +290,9 @@ getAllowedCores <- function() {
     # get cpu allowance from files
     cfs_quota_us <- as.numeric(system("cat /sys/fs/cgroup/cpu/cpu.cfs_quota_us", intern = T))
     cfs_period_us <- as.numeric(system("cat /sys/fs/cgroup/cpu/cpu.cfs_period_us", intern = T))
-    cfs_quota_us / cfs_period_us
+    cores <- floor(cfs_quota_us / cfs_period_us)
+    if(cores < 1){return()}

```

Comment:
is `return()` the same as `return(NULL)` ?

Reply:
Yes.

Reply:
can we have return on another line please
Also run the styler